templates/jobs/main.html

```diff
@@ -0,0 +1,86 @@
+{% extends 'jobs/base.html' %}
+{% load staticfiles %}
+
+{% block content %}
+
+<section id="box-with-picture">
+    <div class="black-canvas">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-8 col-md-offset-2">
+                    <h1>
+                        You've already attended Django Girls event?
+                    </h1>
+
+                    <p>there is much more...</p>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+
+<div class="container">
+    <section id="community">
+        <div class="row">
+            <div class="col-md-9">
+                <h2>Community</h2>
+
+                <p>Here, you will find information about job offers for
+                    juniors, meetups, local groups and other things that will
+                    keep you get going with becoming the fully-fledged Django
+                    programmer.</p>
```

Comment:
I guess we can remove _entry-level_ anytime we need. And Mikey is to review the content, so we can discuss it then. What d'you think?